linode/manager
# chore: [M3-6803] - Use latest `@sentry/react` package and enable performance monitoring

## Description 📝
- Updates our Sentry package and makes sentry more useful

## Major Changes 🔄
- Uses `@sentry/react` rather than `@sentry/browser` because it is better for our use case 📦
- Enables Sentry's performance monitoring 🔎
  - This will duplicate some things New Relic does but it will give us some very nice information that we can all have access to seeing
  - This will give us better observability and debugability

## Preview 📷
![Screenshot 2023-06-28 at 6 15 33 PM](https://github.com/linode/manager/assets/115251059/ae7fba96-0391-4330-942e-c1ea1f4f31f6)

## How to test 🧪
- Set `REACT_APP_SENTRY_URL` in your `packages/manager/.env` (ask me if you need the value)
- Throw an error in a component
- Verify you see the error in Sentry
